### PR TITLE
all runtimes: remove already applied migrations

### PR DESCRIPTION
**Pull request by acatangiu**
Remove migrations already **applied on-chain**.

- [x] Does not require a CHANGELOG entry

### CODE REVIEW

**Comment by ggwpez** [comment]
Looks good, but can we get this in first to have better testing? https://github.com/polkadot-fellows/runtimes/pull/419

File: system-parachains/coretime/coretime-kusama/src/lib.rs [hunk 1]
```diff
@@ -116,12 +116,13 @@ pub type UncheckedExtrinsic =
...
 /// Migrations to apply on runtime upgrade.
 pub type Migrations = (
-	pallet_xcm::migration::MigrateToLatestXcmVersion<Runtime>,
-	pallet_collator_selection::migration::v2::MigrationToV2<Runtime>,
+	// unreleased and/or un-applied
 	cumulus_pallet_xcmp_queue::migration::v5::MigrateV4ToV5<Runtime>,
 	pallet_broker::migration::MigrateV0ToV1<Runtime>,
 	pallet_broker::migration::MigrateV1ToV2<Runtime>,
 	pallet_broker::migration::MigrateV2ToV3<Runtime>,
+	// permanent
```
Comment (bkontur) [inline system-parachains/coretime/coretime-kusama/src/lib.rs:124]: :+1:

**Review by bkontur — APPROVED** [review]

**Review by seadanda — APPROVED** [review]

File: system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs [hunk 2]
```diff
@@ -1071,14 +1071,9 @@ pub type SignedExtra = (
 pub type UncheckedExtrinsic =
 	generic::UncheckedExtrinsic<Address, RuntimeCall, Signature, SignedExtra>;
 
-parameter_types! {
-	pub DmpQueueName: &'static str = "DmpQueue";
-}
-
 /// Migrations to apply on runtime upgrade.
 pub type Migrations = (
-	frame_support::migrations::RemovePallet<DmpQueueName, RocksDbWeight>,
```
Comment (bkontur) [inline system-parachains/asset-hubs/asset-hub-kusama/src/lib.rs:1080]: I think we should be able now to remove `cumulus-pallet-dmp-queue` dependency from everywhere (at least AHK/BHK uses that and/or maybe also remove it from the main Cargo.toml).

**Review by franciscoaguirre — APPROVED** [review]

**Comment by acatangiu** [comment]
> Looks good, but can we get this in first to have better testing? #419

@ggwpez I think this is good to go, yes?

**Review by ggwpez — APPROVED** [review]

**Pull request merged**